src/main.js

```diff
@@ -188,6 +188,13 @@ export class Draggable {
         return !this._mouseOnly && Draggable.supportPointerEvent();
     }
 
+    update({ press = noop, drag = noop, release = noop, mouseOnly = false }) {
```

Comment:
We could avoid duplicating this code if the constructor invokes `update(options)`